ddcommon-ffi/src/option.rs

```diff
@@ -19,6 +19,13 @@ impl<T> Option<T> {
             Option::None => None,
         }
     }
+
```

Comment:
any value in starting to add some tests to this file? I noticed some of the other files in this crate have them now.

Reply:
If you agree maybe I could do it in a separate PR. I'm talking with Daniel about the possibility of porting/moving error handling to the common module so I could add more tests then.

Reply:
No problem. The more I think about it the less concerned I am about a unit test for `as_mut`. If it didn't work correctly then the code wouldn't compile.